solc/CommandLineInterface.cpp

```diff
@@ -668,7 +668,7 @@ void CommandLineInterface::handleCombinedJSON()
 		if (requests.count("srcmap-runtime"))
 		{
 			auto map = m_compiler->runtimeSourceMapping(contractName);
-			contractData["srcmap"] = map ? *map : "";
+			contractData["srcmap-runtime"] = map ? *map : "";
```

Comment:
Shouldn't this be `srcmapRuntime` (as in #763).

Actually I think the `combined-json` output should match the JSON of `jsonCompiler`.